src/dpl/src/CheckPlacement.cpp

```diff
@@ -203,6 +209,42 @@ bool Opendp::overlap(const Cell* cell1, const Cell* cell2) const
          && ll1.getY() < ur2.getY() && ur1.getY() > ll2.getY();
 }
 
+Cell* Opendp::checkOneSiteGaps(Cell& cell) const
+{
+  Cell* gap_cell = nullptr;
+  visitCellBoundaryPixels(
+      cell, true, [&](Pixel* pixel, Edge edge, int x, int y) {
+        Cell* pixel_cell = pixel->cell;
+
+        int abut_x = 0;
+
+        switch (edge) {
+          case Edge::left:
+            abut_x = -1;
+            break;
+          case Edge::right:
+            abut_x = 1;
+            break;
+          default:
+            break;
```

Comment:
It is best to avoid default cases.  If a new enum value is added the compiler can flag any switch statements that need updating.  In this case it reveals that the other values are unneeded.